fabtests/configure.ac

```diff
@@ -37,6 +37,8 @@ AM_CONDITIONAL([MACOS], [test $macos -eq 1])
 AM_CONDITIONAL([LINUX], [test $linux -eq 1])
 AM_CONDITIONAL([FREEBSD], [test $freebsd -eq 1])
 
+AS_IF([test $macos -eq 1], [CPPFLAGS="$CPPLFAGS -I$PWD/include/osx"])
```

Comment:
libfabric has this in Makefile.am, versus changes in configure.ac:
```
if MACOS
common_srcs += src/unix/osd.c
common_srcs += include/osx/osd.h
common_srcs += include/unix/osd.h
endif
```

Reply:
Perfect. Thanks!

Reply:
This didn't work. Best I could do was add `AM_CFLAGS += -I$(srcdir)/include/osx`